Rapid7 Insight IDR New action Close Investigations in bulk (#758)
rapid7/insightconnect-plugins: rapid7_insightidr/help.md

### Actions

#### Advanced Query on Log Set

> This action is used to realtime query into an Insight IDR log set.

Reviewer: ```suggestion
This action is used to realtime query an InsightIDR log set.
```

        "result": "SUCCESS",
        "new_authentication": "false",
        "service": "ntlmssp ",
        "source_json": {
          "sourceName": "Microsoft-Windows-Security-Auditing",
          "insertionStrings": [
            "S-1-0-0",
            "-",
            "-",
            "0x0",
            "X-X-X-XXXXXXXXXXX",
            "user@example.com",
            "example-host",
            "0x204f163c",
            "3",
            "NtLmSsp ",
            "NTLM",
            "",
            "{00000000-0000-0000-0000-000000000000}",
            "-",
            "NTLM V2",
            "128",
            "0x0",
            "-",
            "192.168.50.1",
            "59090"
          ],
          "eventCode": 4624,
          "computerName": "example-host",
          "sid": "",
          "isDomainController": false,
          "eventData": null,
          "timeWritten": "2020-10-02T00:29:13.670722000Z"
        }
      },
      "links": [
        {
          "rel": "Context",
          "href": "https://us.api.insight.rapid7.com/log_search/query/context/xxxx"
        }
      ],
      "sequence_number_str": "123456789123456789"
    }
  ]
}
```

#### Advanced Query on Log

> This action is used to realtime query into Insight IDR logs. 

Reviewer: ```suggestion
This action is used to realtime query InsightIDR logs. 
```

        "result": "SUCCESS",
        "new_authentication": "false",
        "service": "ntlmssp ",
        "source_json": {
          "sourceName": "Microsoft-Windows-Security-Auditing",
          "insertionStrings": [
            "S-1-0-0",
            "-",
            "-",
            "0x0",
            "X-X-X-XXXXXXXXXXX",
            "user@example.com",
            "example-host",
            "0x204f163c",
            "3",
            "NtLmSsp ",
            "NTLM",
            "",
            "{00000000-0000-0000-0000-000000000000}",
            "-",
            "NTLM V2",
            "128",
            "0x0",
            "-",
            "192.168.50.1",
            "59090"
          ],
          "eventCode": 4624,
          "computerName": "example-host",
          "sid": "",
          "isDomainController": false,
          "eventData": null,
          "timeWritten": "2020-10-02T00:29:13.670722000Z"
        }
      },
      "links": [
        {
          "rel": "Context",
          "href": "https://us.api.insight.rapid7.com/log_search/query/context/xxxx"
        }
      ],
      "sequence_number_str": "123456789123456789"
    }
  ]
}
```

#### Get All Logs

> This action is used to request a list of all Logs for an account. This action should be used when querying multiple related services. 

Reviewer: ```suggestion
This action is used to request a list of all logs for an account. This action should be used when querying multiple related services. 
```

### Actions

#### Close Investigations in Bulk

> This action is used to close all investigations that match the parameters.

Reviewer 2: Can you be more specific than "match the parameters"

Is it: 
```suggestion
This action is used to close all investigations that fall in a date range.
```

> This action is used to close all investigations that fall in a date range.

Reviewer 3: ```suggestion
This action is used to close all investigations that fall within a date range.
```

##### Input

|Name|Type|Default|Required|Description|Enum|Example|
|----|----|-------|--------|-----------|----|-------|
|alert_type|string|None|False|The category of alerts that should be closed|None|Account Created|
|datetime_from|string|None|False|An ISO formatted timestamp, default last week|None|2018-07-01 00:00:00|
|datetime_to|string|None|False|The category of alerts that should be closed, when empty use now|None|2018-07-01 00:00:00|

> |max_investigations_to_close|integer|None|False|The category of alerts that should be closed|None|10|

Reviewer 2: The variable name doesn't match the description. 

Is this a category or a limit?
